index.html

```diff
@@ -1,29 +1,195 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.cdnfonts.com/css/anurati" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.typekit.net/xse6ume.css">
+    <title>Assignment 2</title>
+</head>
+<body>
+    <!-- Header -->
+    <div class="header">
+        <h1>OUR ECOLOGY</h1>
+        <p><a href="#">Home</a></p>
+        <p><a href="#mission">Mission</a></p>
+        <p><a href="#">Community</a></p>
+        <p><a href="#news">News</a></p>
+        <p><a href="#">About Us</a></p>
+        <p><a href="#contact">Contact</a></p>
+        <p><a href="#donate">Donate</a></p>
+        <!-- Search bar text and icon -->
+            <a><input type="text" placeholder="Search"></a>
+                <img src="search.png" alt="Search icon" width="30">
+    </div>
 
-    <!-- Set the screens default zoom / scale to 1 -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <!-- Specifies the document mode for Internet Explorer -->
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Hero -->
+    <div class="hero">
+        <h1>MAKE A CHANGE TO THE <br>
+            ENVIRONMENT TODAY.</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <br>
+            tempor incididunt ut labore et dolore magna aliqua. Et ligula ullamcorper <br>
+            malesuada proin libero. Platea dictumst vestibulum rhoncus est pellentesque <br>
+            elit ullamcorper dignissim cras. Eget nullam non nisi est sit amet facilisis <br>
+            magna etiam. Quam adipiscing vitae proin sagittis nisl rhoncus mattis <br>
+            rhoncus urna. Est placerat in egestas erat imperdiet sed euismod nisi. Blandit <br>
+            volutpat maecenas volutpat blandit aliquam etiam erat velit scelerisque. Ipsum <br>
+            consequat nisl vel pretium lectus quam id leo. Donec massa sapien faucibus <br>
+            et molestie. Interdum velit laoreet id donec ultrices tincidunt arcu. Eu sem <br>
+            integer vitae justo eget magna. Eget mauris pharetra et ultrices neque. <br>
+        </p>
 
-    <!-- We are using a CSS Reset file to standardize browser default css -->
-    <link rel="stylesheet" href="reset.css" />
+        <!-- Donate button text -->
+        <div class="donate" id="donate">
+            <p><a href="#">DONATE</a></p>
```

Comment:
Try to avoid classes and IDs that have the same name. You may run into collisions, especially when we start to use JavaScript and the `getElementById()` method.